lib/persistence/object.rb

```diff
@@ -11,15 +11,6 @@ class Object
       serviceproducts
     )
 
-    # Current FL clients:
-    #   dcdistribution
-    #   oilsolutionsgroup
-    #   unionville
-    #   kidmademodern
-    #   justsprinklers
-
-    CONNECTION_IDS_TO_LOG = %w()
```

Comment:
You still need this, the ENV file cannot be an array. It will be a space or comma separated list, you'll need to split it.

`CONNECTION_IDS_TO_LOG = ENV.fetch(IDS_TO_LOG, '').split(',')` if IDS_TO_LOG equals "osg,kidmademodern"

`fetch` will ensure that if the env var is not set, an empty string will be returned and thus split won't break.

Reply:
Whoops I see you're splitting down below. But having the split done once seems better in case we keep fiddling with this. Maybe even a method `should_log?(:connection_id)` ?

Reply:
Yeah this is tricky - we're ONLY using this logic for a couple methods, not the whole class or project. So I feel like I need to do some renaming to make it clearer. Maybe a method would be better `should_log_s3_obj_movement?(id)`

Reply:
It sounds like a mouthful right now, but will be very helpful three months from now, when we are scratching our heads trying to remember what does that method do. :-)